src/widgets/ActivityGroups/ui/ActivityCard/index.tsx

```diff
@@ -101,20 +102,45 @@ export const ActivityCard = ({ activityListItem }: Props) => {
 
   const restartActivity = () => {
     onStartActivity(true);
-    Mixpanel.track(MixEvents.ActivityRestarted, { [MixProperties.AppletId]: context.applet.id });
+
+    const analyticsPayload: MixpanelPayload = {
+      [MixpanelProps.AppletId]: context.applet.id,
+      [MixpanelProps.ActivityId]: activityListItem.activityId,
+    };
+
+    if (isFlow) {
+      analyticsPayload[MixpanelProps.ActivityFlowId] = activityListItem.flowId;
+    }
+
+    Mixpanel.track(MixpanelEvents.ActivityRestarted, analyticsPayload);
   };
+
   const resumeActivity = () => {
     onStartActivity(false);
-    Mixpanel.track(MixEvents.ActivityResumed, { [MixProperties.AppletId]: context.applet.id });
+
+    const analyticsPayload: MixpanelPayload = {
```

Comment:
We don't seem to do anything with this variable. Should it be removed? The old `ActivityResumed` event was only tracking applet ID

Reply:
I just missed passing this value to the `Mixpanel.track` call – good catch!

Reply:
I was wondering if we should be tracking activity ID and Flow ID for the `ActivityResumed`, since they're not mentioned in the ticket and the old one wasn't tracking them. It makes sense of course, but I thought it was something to be noted